src/http_rest_api_handler.cpp

```diff
@@ -325,4 +326,36 @@ Status HttpRestApiHandler::processModelStatusRequest(
     return StatusCode::OK;
 }
 
+Status HttpRestApiHandler::processModelControlApiRequest(std::string& response) {
+    SPDLOG_INFO("ModelControlApi flow triggered.");
+    Status status;
+    auto& config = ovms::Config::instance();
+    auto& manager = ModelManager::getInstance();
+    bool isConfigFileReloadNeeded = manager.configFileReloadNeeded();
+    if (isConfigFileReloadNeeded) {
+        status = manager.loadConfig(config.configPath());
+        if (!status.ok()) {
+            return status;
+        }
+    }
```

Comment:
What if config didn't change? Don't we want to scan filesystem for changes in models versions?
In case user didn't set periodic filesystem scans.